MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+The current maintainers of this repo are [@adamralph](https://github.com/adamralph), [@bording](https://github.com/bording), and [@WojcikMike](https://github.com/WojcikMike).
+
+The maintainers [watch](https://github.com/Particular/NServiceBus.RabbitMQ/watchers) this repo and undertake the following responsibilities:
```

Comment:
Looking at the list of responsibilities below, the wording feels like it might be off. Instead of "Ensuring", "Releasing", and "Reviewing", should it be "Ensure", Release", and "Review"?